bundles/org.openhab.automation.jsscripting/README.md

```diff
@@ -21,23 +36,212 @@ log:set DEBUG org.openhab.core.automation
 
 For more information on the available APIs in scripts see the [JSR223 Scripting]({{base}}/configuration/jsr223.html) documentation.
 
-## Script Examples
+The following examples show how to access common openHAB functionalities.
+
+## Logging
 
-JavaScript scripts provide access to almost all the functionality in an openHAB runtime environment.
 As a simple example, the following script logs "Hello, World!".
 Note that `console.log` will usually not work since the output has no terminal to display the text.
 The openHAB server uses the [SLF4J](https://www.slf4j.org/) library for logging.
 
-```js
-const LoggerFactory = Java.type('org.slf4j.LoggerFactory');
-
-LoggerFactory.getLogger("org.openhab.core.automation.examples").info("Hello world!");
+```javascript
+let logger = Java.type('org.slf4j.LoggerFactory').getLogger('org.openhab.rule.' + this.ruleUID);
```

Comment:
Where does "ruleUID" come from? I don't think that this is available via `this`. Sounds like it may need to be added, although it would probably be available via an import, like `require("@current-script").ruleUID` or something.

Reply:
I don‘t know where it comes from, but it works that way.

Reply:
When you say it works, you mean that you have tested it with this runtime? I don't understand how it would actually work.

Reply:
Yes, I have tested it with the 3.2.0.M4 Milestone and it works.